staging/src/k8s.io/apimachinery/pkg/runtime/serializer/protobuf/testing/types.go

```diff
@@ -0,0 +1,17 @@
+package testing
+
+import "k8s.io/apimachinery/pkg/runtime"
+
+// +k8s:protobuf-gen=package
+
+// TypeA
+// +protobuf=true
+type TypeA struct {
```

Comment:
Do we need to introduce new types really?

Can't we reuse existing ones, e.g.:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/apis/testapigroup/v1/types.go#L33

Reply:
oh, I missed `testapigroup` package, no need to intro a new type then.

Reply:
Please remove then - it will make this PR much smaller (and thus easier to review).

Reply:
done.